tests/mmm/test_fourier.py

```diff
@@ -272,3 +279,83 @@ def test_change_name() -> None:
 def test_serialization_to_json() -> None:
     fourier = YearlyFourier(n_order=2)
     fourier.model_dump_json()
+
+
+@pytest.fixture
+def yearly_fourier() -> YearlyFourier:
+    prior = Prior("Laplace", mu=0, b=1, dims="fourier")
+    return YearlyFourier(n_order=2, days_in_period=365, prior=prior)
+
+
+@pytest.fixture
+def monthly_fourier() -> MonthlyFourier:
+    prior = Prior("Laplace", mu=0, b=1, dims="fourier")
+    return MonthlyFourier(n_order=2, days_in_period=30, prior=prior)
```

Comment:
Why is `days_in_periods` being redefined here?

Reply:
+1

Reply:
sure, no worries. I noticed you have constants for this, but it should work without it too.

Reply:
I'll push something tonight!

Reply:
Only `# noqa: UP038` remains in the latest commit; have removed` <3.10` comment. Hope that works, if anything else let me know please. 

and yeah `days_in_period` from `test_fourier.py`  has been removed